exercises/knapsack/description.md

```diff
@@ -0,0 +1,10 @@
+Given a set of items, each with a weight, w and a value, v, determine the number of each item to include in a collection so that the total weight is less than or equal to a given limit and the total value is as large as possible
```

Comment:
this sentence sort of implies the general knapsack problem to me because "number of each item" hasn't told me that it's restricted to 1 and 0. It seems to me like that is an important aspect of the problem, and that if we can express it here, we should do so to help students understand what they have to plan for.

Reply:
Agreed, I have tweaked the description and hopefully made it more interesting.